core/src/apps/cardano/protocol_magics.py

```diff
@@ -0,0 +1,12 @@
+class ProtocolMagics:
```

Comment:
any reason this is a class as opposed to either a static dict (doesn't seem to be too big), or just a function (if RAM usage is a concern)?

Reply:
We've made it into a class for convenience.

Is something like this what you had in mind?
```
PROTOCOL_MAGICS = {"mainnet": 764824073, "testnet": 42}


def protocol_magic_to_ui_string(protocol_magic: int) -> str:
    ui_string_map = {
        PROTOCOL_MAGICS["mainnet"]: "Mainnet",
        PROTOCOL_MAGICS["testnet"]: "Testnet",
    }

    return ui_string_map.get(protocol_magic, "Unknown")
```

Reply:
Updated in this [commit](https://github.com/trezor/trezor-firmware/pull/1096/commits/71f2191b6f0a3c4ecea9d0ded1a2f62b814283a4).

Reply:
more like:
```
MAINNET = 764824073
TESTNET = 42

NAMES = {
	MAINNET: "Mainnet",
	TESTNET: "Testnet",
}

def to_ui_string(value: int) -> str:
	return NAMES.get(value, "Unknown")
```

for ease of use. then you can refer to `protocol_magics.MAINNET` for the constant, `protocol_magics.to_ui_string(xyz)` for the query

Reply:
basically you're using a class to do a module's job, just move everything you had before to top-level

Reply:
Ok, I got it. Thanks! Updated [here](https://github.com/trezor/trezor-firmware/pull/1096/commits/f4a1990e9e8dbf1721314eba69aed8e4d0e535e4).